sys-fs/ecryptfs-utils/metadata.xml

```diff
@@ -2,6 +2,14 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<!-- maintainer-needed -->
```

Comment:
Please remove the `<!-- maintainer-needed -->` as well, otherwise this package will still pop up as a package needing a maintainer

Reply:
And you should also sign-off your commits, if you set this in make.conf:
```
PORTAGE_GPG_KEY='KEY-FINGERPRINT'
DCO_SIGNED_OFF_BY='Your Real Name <someone@example.com>'
```
And use `repoman ci` for committing, your commits will be singed-off automatically.

Reply:
Well there are few things here, 
first of all we don't let people "hoard" ownership without actual contributions to a package. 
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Taking_over_an_existing_package part 3. We really want to see what you're capable of before becoming a maintainer.

Then as @AndrewAmmerlaan has just pointed out, this commit is not signed off properly, we need that sign-off so we can merge your contribution to ::gentoo tree. 
https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

Also in future please use our commit summary syntax, 
https://www.gentoo.org/glep/glep-0066.html#commit-messages (this'd be an easy fix while merging, but make our lives easier by getting it done correctly in the first place)

Reply:
And just to be clear, we **do** accepts contributions to maintainer-needed packages **without** you having to become a maintainer for them. So please help us out by keep fixing maintainer-needed packages.